x-pack/plugins/ml/public/application/jobs/jobs_list/components/utils.js

```diff
@@ -240,7 +241,12 @@ export async function cloneJob(jobId) {
       return;
     }
 
-    if (cloneableJob !== undefined && originalJob?.custom_settings?.created_by !== undefined) {
+    const createdBy = originalJob?.custom_settings?.created_by;
```

Comment:
When I export jobs, looks like `created_by` is removed (tried an advanced job and a multi-metric one). I can't remember - was this intentional?

Reply:
We're using the flag `exclude_generated: true` when retrieving the jobs from es. This removes the `created_by` prop.
If we think it would be better to not strip this out, I can change it in a separate PR.

Reply:
Sounds a good idea to retain `created_by` on export, and to do it in a separate PR.